guides/common/modules/proc_preparing-for-disaster-recovery-with-two-active-project-servers.adoc

```diff
@@ -0,0 +1,44 @@
+[id="preparing-for-disaster-recovery-with-two-active-project-servers"]
+= Preparing for disaster recovery with two active {ProjectServer}s
+
+Create a second {ProjectServer} by restoring a backup of your first {ProjectServer}.
+Configure both servers to operate independently in their respective data centers, but ensure that their content, such as repositories or host configuration, does not drift apart over time.
+
+.Procedure
+. Back up your {ProjectServer}.
+For more information, see xref:backing-up-{project-context}-server-and-{smart-proxy-context}_admin[].
+. Restore the backup on a system that will serve as your other {ProjectServer}.
+For more information, see xref:restoring-{project-context}-server-or-{smart-proxy-context}-from-a-backup_admin[].
++
+[NOTE]
+====
+Each server must have a distinct hostname and IP address.
+This enables you to re-register hosts if one of the servers fails.
+====
+. Ensure that content on your servers remains consistent:
+** If you want both servers to act as sources for synchronization and content view creation, follow these guidelines to prevent content drift:
+*** Regularly synchronize repositories on both servers.
+For more information, see {ContentManagementDocURL}[Synchronizing repositories] in _{ContentManagementDocTitle}_.
+*** Ensure that content views on both servers match.
+// *** What else?
+** If you want only one server to manage synchronization and content views and let the other server synchronize its content from the first server, configure Inter-Server Synchronization (ISS).
```

Comment:
A strategy we can mention here is to have a single source of truth that is only handling content view definitions, and then each server is syncing from it. This adds complexity but it splits the roles up and can help:

```
     |--- Satellite for CVs only ---|
               /                               \
             /                                   \
           /                                       \
|--Satellite for Hosts --|     |--Satellite for hosts--|
```

Reply:
Ok, I did my best and added this as a third option.